apis/cc-infix-en_GB/atrium-api-cc-infix-en_GB-jvm/src/test/kotlin/ch/tutteli/atrium/api/cc/infix/en_GB/CharSequenceContainsAtLeastAssertionsSpec.kt

```diff
@@ -29,9 +33,9 @@ class CharSequenceContainsAtLeastAssertionsSpec : ch.tutteli.atrium.spec.integra
 
         private fun containsAtLeast(plant: Assert<CharSequence>, atLeast: Int, a: Any, aX: Array<out Any>): Assert<CharSequence> {
             return if (aX.isEmpty()) {
-                plant to contain atLeast atLeast value a
+                (plant.asExpect().contains(o)).atLeast(atLeast) value a
```

Comment:
Did the replacements actually work for you? You defined them correctly IMO.
They did not work for me (created a Kotlin issue https://youtrack.jetbrains.com/issue/KT-38496)